CPS-XXXX/README.md

```diff
@@ -0,0 +1,82 @@
+---
+CPS: ?
+Title: Ouroboros not accounting for typical global network delays causes centralisation
+Status: Open
+Category: ?
+Authors: Terminada
```

Comment:
```suggestion
Authors:
  - Terminada
```
In our standard format, the content of fields that might contain multiple values must either be an empty set or an unordered list.